brand/models/commentary.py

```diff
@@ -19,7 +19,7 @@ class Commentary(models.Model):
     # Metadata
     brand = models.OneToOneField(
```

Comment:
Should probably rename this class bank commentary since insurance brands may be in the works later on?

Reply:
I can do this (it'll require a bit of refactoring), but I don't think this is a useful line of thinking at this moment:
- refactoring a model name is easy
- if this app is extended include insurances ,there'll be a bunch more changes to be made anyway
- most importantly: including insurance is nowhere on any roadmap, both pre-launch and post-launch